### ISSUE TITLE
JAWS remains quiet as the cursor moves through input fields in Zotero desktop

### ISSUE THREAD

**Issue by abaevbog** [FreedomScientific/standards-support#830]
### Summary

Jaws does not announce the content of input or textarea fields as the cursor moves between characters in [Zotero](https://github.com/zotero/zotero) desktop app. Either nothing is announced, or “blank”. 

### Steps to reproduce:
1. Download the last beta build of zotero https://www.zotero.org/support/beta_builds
2. Input “test” in the visible search field “All Fields & Tags”
3. Move cursor to the beginning of the field with left arrow

### Expected result

Jaws will announce characters as the cursor moves through the input field

### Actual result

Jaws does announce the characters that were just typed - “test” - and is then quiet if you move the cursor back and forth with arrow keys.

### Example

[Screen recording of the issue](https://www.dropbox.com/scl/fi/799co4s5cwoyaebmy06xk/jaws_input_issue.mov?rlkey=4680pab13c54ya6024kbmx8ea&st=qxrx4y32&dl=0)

### Additional Information

- Zotero is a mozilla-based desktop application (like firefox or thunderbird), so we have a mixture of XUL and HTML. Oddly enough, I could not reproduce this in Firefox itself. JAWS announces text just fine inside of the search field of downloads window (Options > Downloads) which is essentially the same [search-text component](https://searchfox.org/mozilla-central/source/browser/components/places/content/places.xhtml#339) inside of a XUL window.

- This seems to happen only to input fields that are directly inside of a XUL window. E.g. inputs inside of an iframe are not affected as seen in the screen recording above:
   - Click the “New note” button (4th button in the middle toolbar)
   - Select “New standalone note” - notice a new note appear in the right pane
   - Click on the magnifying glass in the right pane and type “test” in the appearing search field.
   - The note editor is an iframe. If cursor moves within that field, everything works as expected. 

- Old issue this sounds somewhat similar to https://github.com/FreedomScientific/standards-support/issues/62

- Another mozilla-based open source project where I observed similar behavior: https://github.com/RealityRipple/Pale-Moon

- To examine the Zotero DOM, one can use Firefox dev tools:

  - Run zotero with debugger enabled `C:\Program Files\Zotero> .\zotero.exe -debugger`
  - Go to `about:debugging` in Firefox esr 115
  - Add localhost:6100 as a network location and connect to it
  - Click on “Inspect” next to “Multiprocess Toolbox” under "Processes" tab at the bottom

- `aria-describedby` set on inputs gets ignored by JAWS as well

Due to the fact that I did not observe this in Firefox, the issue may be in some configuration on our end. However, JAWS is the only screen reader that is having such issues (NVDA and Voiceover on mac do not), so any advice or guidance on how this could be addressed is appreciated!

#### JAWS version and build number

JAWS 2024.2403.3
 
#### Operating System and version

Tested on Windows 11 PRO 23H2 VM

#### Browser and version:

Most current Zotero beta build is based on Firefox 115
